tests/cloudpickle_test.py

```diff
@@ -507,6 +507,27 @@ def test_cell_manipulation(self):
             msg='cell contents not set correctly',
         )
 
+    def test_logger(self):
+        logger = logging.getLogger('cloudpickle.dummy_test_logger')
+        self.assertTrue(pickle_depickle(logger) is logger)
+
+        dumped = cloudpickle.dumps(logger)
+
+        code = """if 1:
```

Comment:
`textwrap.dedent` is a bit more annoying to use (you have to make sure the first line has the same indent as the other ones, which this idiom doesn't require).